rdf-delta-fuseki/pom.xml

```diff
@@ -47,6 +47,12 @@
       <artifactId>jena-fuseki-main</artifactId>
     </dependency>
 
+    <!-- Included so that full text search will be possible -->
```

Comment:
That would work though other inclusions happen in the next step in  `rdf-delta-fuseki-server` where the combined jar executable is built. For example, that is where the logging implementation is put in. This module (`rdf-delta-fuseki`) is the extra java code for the Fuseki-Delta features.

Reply:
yes, originally, I had placed the dependency in `rdf-delta-fuseki-server` because jena-text is only needed for the fuseki server. However, the jena-text code still wasn't being found at runtime. Do you have a suggestion as to why this may be the case or how it might be resolved while still having the dependency in `rdf-delta-fuseki-server`?

Reply:
Weird.

Are you running the maven-built server or running an embedded Fuseki by depending on rdf-delta-fuseki?

If running the maven-built server, did you run the whole build? In my experience, maven can get confused if the parent isn't in the same build run as the server artifacts.

I tried with a dependency in rdf-delta-fuseki-server and the jar is about 30M with Lucene and 24M without (`ls -l rdf-delta-fuseki-server/target/rdf-delta-fuseki-server-0.9.1-SNAPSHOT.jar`). Is that what you see?

The other possible problem area is the services resources but looking in the jar (META-INF/services/org.apache.jena.sys.JenaSubsystemLifecycle) includes `org.apache.jena.query.text.InitJenaText` which is correct.

Reply:
I built the project, then used the `dcmd` cli that was generated from that: `dcmd fuseki --config /fuseki/configuration/$FUSEKI_CONFIG.ttl`

Reply:
Ah! `dcmd`. 

`dcmd` does not use the packaged "server" artifact but the `rdf-delta-fuseki` artifact.

What I am trying to avoid is for every use of delta+fuseki to pull in Lucene. That isn't really so bad in this case (it adds ~6M) but the principle is that Fusek main, with Delta services, is something to add jars to. `jena-text` isn't the only case - custom local SPARQL functions would need to add their jar+dependencies.

Adding `jena-text` to `dcmd` is fine - `dcmd` is a representative of one of those combined systems. That leaves depending on the delta+fuseki artifact minimal and clean.

Reply:
Awesome! I was thinking the same thing, don't want to pull in all of those libs everytime if they aren't needed. I'll make the changes.

Reply:
Thank you for your patience. Making the choices here is somewhat permanent as other people then rely on it. If it's sole user setup, it can be whatever works!

Reply:
Ok, I've made the updates, hopefully I got them in the right spot. 🤞